ui/src/pages/DevelopmentalMilestones.jsx

```diff
@@ -0,0 +1,402 @@
+import {
+  Container,
+  Stack,
+  Button,
+  Grid,
+  Snackbar,
+  Typography,
+  Divider,
+  useMediaQuery,
+  CircularProgress,
+  Modal,
+} from "@mui/material";
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import Tab from "@mui/material/Tab";
+import TabContext from "@mui/lab/TabContext";
+import TabList from "@mui/lab/TabList";
+import TabPanel from "@mui/lab/TabPanel";
+import { Box } from "@mui/material";
+import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import CurrentPatient from "../components/CurrentPatient";
+import { apiHost, createEncounter, FhirApi } from "../lib/api";
+import { useFormik } from "formik";
+import * as yup from "yup";
+import Preview from "../components/Preview";
+import FormFields from "../components/FormFields";
+import developmentalMilestonesFields from "../lib/forms/developmentalMilestones";
+import { getSections } from "../lib/getFormSections";
+
+export default function DevelopmentalMilestones() {
+  let [visit, setVisit] = useState();
+  let navigate = useNavigate();
+  let [open, setOpen] = useState(false);
+  let [loading, setLoading] = useState(false);
+  let [message, setMessage] = useState(false);
+  let [observations, setObservations] = useState([]);
+  let isMobile = useMediaQuery("(max-width:600px)");
+  const [newVisit, setNewVisit] = useState(false);
+  let [DevelopmentalMilestonesEncounters, setDevelopmentalMilestonesEncounters] = useState(
+    []
+  );
+  const handleClose = () => setOpenModal(false);
+  const handleOpen = () => setOpenModal(true);
+  const [value, setValue] = useState("1");
+  let [openModal, setOpenModal] = useState(false);
+
+  const [inputData, setInputData] = useState({});
+  const [preview, setPreview] = useState(false);
+
+  const fieldValues = Object.values(getSections(developmentalMilestonesFields, DevelopmentalMilestonesEncounters.length, DevelopmentalMilestonesEncounters.length+1)).flat();
```

Comment:
Not a deal breaker, but try and change all variables to a camelCase in cases where they are capitalized